server/models/Users.js

```diff
@@ -96,12 +106,25 @@ class Users {
   }
 
   removeUser(username, callback) {
-    this.db.remove({username: username}, {}, (err, numRemoved) => {
+    this.db.findOne({username: username}).exec((err, user) => {
```

Comment:
Do we need to call `findOne` first?

Reply:
we need the `user._id` in order to remove the db directory